platforms/win32/plugins/FilePlugin/sqWin32FilePrims.c

```diff
@@ -37,6 +37,7 @@ extern struct VirtualMachine *interpreterProxy;
 
 #define true  1
 #define false 0
+static const DWORD MAX_DWORD = 4294967295;
```

Comment:
```suggestion
static const DWORD MAX_DWORD = MAXDWORD;
```

Reply:
...so really one could just use the constant provided by the Windows headers below.